external_resources/api.py

```diff
@@ -55,9 +58,52 @@ def is_external_url_broken(
     return is_url_broken(url)
 
 
-def is_backup_url_broken(
-    external_resource: WebsiteContent,
-) -> tuple[bool, Optional[int]]:
-    """Check if backup url of the provided WebsiteContent is broken"""
-    url = external_resource.metadata.get("backup_url", "")
-    return is_url_broken(url)
+def submit_url_to_wayback(
+    url: str,
+) -> Optional[str]:
+    """
+    Submit the external resource URL to the Wayback Machine and
+    return the response
+    """
+    if not url:
+        return None
```

Comment:
If you do wanna keep this check, we should return the same  data type (i.e., `{}`) from this method

Reply:
Removing this check.